packages/react-core/src/components/InlineEdit/InlineEditToggle.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styles from '@patternfly/react-styles/css/components/InlineEdit/inline-edit';
+import { css } from '@patternfly/react-styles';
+import { Button, ButtonProps, ButtonVariant } from '../Button';
+import PencilAltIcon from '@patternfly/react-icons/dist/esm/icons/pencil-alt-icon';
+
+/** An edit button that must be displayed when the inline edit's edit mode is not enabled. This should be
+ * passed to the inline edit component's customEditToggle property when more customization is desired.
+ */
+
+export interface InlineEditToggleProps extends React.HTMLProps<HTMLDivElement> {
+  /** Text content rendered inside the "edit" button, in addition to the edit icon. */
+  children?: string;
+  /** Adds an accessible name to the "edit" button. This must be passed in when the edit button does not
```

Comment:
I would remove the "must be passed in", because there will always be a default 'Enable edit mode' `ariaLabel`.